apps/emqx_exhook/test/emqx_exhook_SUITE.erl

```diff
@@ -1,5 +1,5 @@
 %%--------------------------------------------------------------------
-%% Copyright (c) 2020-2023 EMQ Technologies Co., Ltd. All Rights Reserved.
+%% Copyright (c) 2020-2024 EMQ Technologies Co., Ltd. All Rights Reserved.
```

Comment:
Auto-update copyright hook... 
This change is accidental, but I guess it won't hurt.